Libraries/Components/TextInput/TextInput.js

```diff
@@ -517,8 +513,8 @@ var TextInput = React.createClass({
         style={[this.props.style]}
         autoCapitalize={autoCapitalize}
         autoCorrect={this.props.autoCorrect}
-        textAlign={textAlign}
-        textAlignVertical={textAlignVertical}
+        textAlign={this.props.textAlign}
```

Comment:
I think I'd still prefer this to be moved into styles as opposed to being a view prop. Just for the sake of consistency with the Text element.

Note that currently `style` property of `TextInput` element use the same `propType` as `Text` (see [here](https://github.com/facebook/react-native/blob/c11dce36e6488c77eb16c121ea7cfe497c26ff4a/Libraries/Components/TextInput/TextInput.js#L314)), which allows `textAlign` (see [here](https://github.com/facebook/react-native/blob/62e8ddc20561a39c3c839ab9f83c95493df117c0/Libraries/Text/TextStylePropTypes.js#L40)). This means that using `textAlign` for `TextInput` in `style` won't result in a warning.

I think we should add `textAlignVertical` to the `style` definition (with an annotation it's android-only) and remove it from `propTypes` definition for `TextInput`